common/ellipses.js

```diff
@@ -19,6 +19,7 @@
                     fromTuple: "=?"
                 },
                 link: function (scope, element) {
+                    // TODO: Wrap most of the body of this function in a try/catch with generic catchAll
```

Comment:
`init()` should be wrapped with `catchWrapper()`.
`var init = Utils.catchWrapper(function () {...});`